backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/views/purchase_prices_in_stock_uom_plv_v.sql

```diff
@@ -38,10 +38,13 @@ SELECT pl.m_pricelist_id,
                    ),
                0
            ) AS ProductPriceInStockUOM,
-       (SELECT p.c_uom_id FROM m_product p WHERE pp.m_product_id = p.m_product_id) AS C_UOM_ID
+       p.c_uom_id
 FROM cte,
      m_productprice pp
          JOIN m_pricelist_version plv ON plv.m_pricelist_version_id = pp.m_pricelist_version_id
          JOIN m_pricelist pl ON plv.m_pricelist_id = pl.m_pricelist_id AND pl.issopricelist = 'N'
+         LEFT JOIN m_product p ON p.m_product_id = pp.m_product_id AND p.isactive = 'Y' AND p.ispurchased = 'Y' AND p.discontinued = 'N'
```

Comment:
why `LEFT JOIN M_Product` and not `INNER JOIN`?
also why do you want to have null C_UOM_ID in case the product is not active or not purchased

same question for m_pricelist_version and m_pricelist?

Reply:
copied wrong version and join = inner join